storage.bs

```diff
@@ -447,7 +450,7 @@ as persistent without involvement from the <a for=/>origin</a> or user. This mak
 useful for resources the user needs to have available while offline or resources the user creates
 locally.
 
-<p>The <dfn for="PermissionName" enum-value>"<code>persistent-storage</code>"</dfn>
+<p>The {{"persistent-storage"}}
```

Comment:
Thanks! Can you add `PermissionName/` in front here and elsewhere?

Reply:
One other request, when doing this, can you put the quotes outside the `{{...}}` as in `"{{...}}"`. That matches the way we format things better.

Reply:
Done.